web/src/admin/property-mappings/BasePropertyMappingForm.ts

```diff
@@ -1,11 +1,60 @@
+import { docLink } from "@goauthentik/common/global";
+import { CodeMirrorMode } from "@goauthentik/elements/CodeMirror";
 import { ModelForm } from "@goauthentik/elements/forms/ModelForm";
 
 import { msg } from "@lit/localize";
+import { TemplateResult, html } from "lit";
+import { ifDefined } from "lit/directives/if-defined.js";
+
+interface PropertyMapping {
+    name: string;
+    expression: string;
+}
+
+export abstract class BasePropertyMappingForm<T extends PropertyMapping> extends ModelForm<
+    T,
+    string
+> {
+    docLink(): string {
```

Comment:
This is needed for the changes in #10652. I'll update that PR once this one gets merged, so we can have different docs for provider/source.